sle15/profiles/standard.profile

```diff
@@ -4,8 +4,82 @@ title: 'Standard System Security Profile for SUSE Linux Enterprise 15'
 
 description: |-
     This profile contains rules to ensure standard security baseline
-    of a SUSE Linux Enterprise 15 system. Regardless of your system's workload
-    all of these checks should pass.
+    of a SUSE Linux Enterprise 15 system. Includes recommendations based on SUSE
+    Hardening Guide
```

Comment:
Let's update the description to the following:

```
    This profile contains rules to ensure standard security baseline
    of a SUSE Linux Enterprise 15 system based off of the SUSE 
    Hardening Guide. Regardless of your system's workload all of
    these checks should pass.
```

Reply:
Complete